features/solvers.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Check for solvers available in Thoth
+    Scenario: Check there are minimum solvers available in Thoth
+        Given a set of solvers
+            | name |
```

Comment:
@pacospace can we use `solver_name` to make it similar to the response body:
```json
{
  "parameters": {},
  "solvers": {
    "python": [
      {
        "os_name": "rhel",
        "os_version": "8",
        "python_version": "3.6",
        "solver_name": "solver-rhel-8-py36"
      },
      {
        "os_name": "fedora",
        "os_version": "31",
        "python_version": "3.8",
        "solver_name": "solver-fedora-31-py38"
      },
      {
        "os_name": "fedora",
        "os_version": "31",
        "python_version": "3.7",
        "solver_name": "solver-fedora-31-py37"
      }
    ]
  }
}
```

Reply:
I totally agree, thanks @bissenbay